components/sections/CompanyOverview/index.jsx

```diff
@@ -0,0 +1,109 @@
+import { useEffect, useState } from "react";
+import ArrowIcon from "../../ui/ArrowIcon";
+import Overview from "./Overview";
+import { motion as Motion } from "framer-motion";
```

Comment:
Renaming `motion` is not a great idea, cause it's not a component however it's an object that contains components. Or you can rename it by restructuring.

```suggestion
import { motion } from "framer-motion";
```

then 

```js
  const {div : MotionDiv} = motion.div
```